samples/routing/fair-distribution/Core_6/Client/Program.cs

```diff
@@ -83,9 +83,9 @@ static void AddRouting(EndpointConfiguration endpointConfiguration)
         #region Routing
 
         const string server = "Samples.FairDistribution.Server";
-        var routing = endpointConfiguration.Routing();
+        var routing = endpointConfiguration.UseTransport<MsmqTransport>().Routing();
         routing.RouteToEndpoint(typeof(PlaceOrder), server);
-        routing.Mapping.Physical.Add(
+        routing.GetSettings().GetOrCreate<EndpointInstances>().Add(
```

Comment:
the goal was separate these more advanced APIs from the basic API so it does not confuse users which just try to setup basic routes. It does of course not prevent users from hard coding it, but we wanted to prevent them from going "ah, I'll just use this "RouteToAddress" overload here. We're very open to suggestions.